packages/clients/dish/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+## unpublished
+
+- Fix: The client now properly throws an error if `queryParameters` is missing on the `searchMethod` configuration of the `dish` search.
```

Comment:
I don't think this is useful information on the DISH client. We're doing the full configuration ourselves here. From any customer perspective, nothing changed, and this doesn't require releasing.

Reply:
Removed the entry